test/util/bspNet/waits.ts

```diff
@@ -112,10 +112,19 @@ export const waitForBspVolunteerWithoutSealing = async (
  *
  * @throws Will throw an error if the expected extrinsic or event is not found.
  */
-export const waitForBspStored = async (api: ApiPromise, checkQuantity?: number) => {
+export const waitForBspStored = async (api: ApiPromise, checkQuantity?: number, bspId?: string) => {
+  // TODO: add bsp ID to check if is not confirming storing
   // To allow time for local file transfer to complete (10s)
   const iterations = 100;
   const delay = 200;
+
```

Comment:
use `invariant(<condition>,<error message>)` as not only is it neater but also provides type-narrowing on subsequent code using vars in the condition